tests/Unit/Console/Commands/ServeCommandTest.php

```
@@ -85,4 +86,258 @@ protected function handleServerExecution(array $serverData): void
+                ];
+            }
+        };
+
+        $publicDir = getcwd() . DS . 'public';
+        $indexFile = $publicDir . DS . 'index.php';
+        if (!is_dir($publicDir)) {
+            mkdir($publicDir, 0777, true);
+        }
+        file_put_contents($indexFile, "<?php echo 'ok';");
+
+        $socket = stream_socket_server('tcp://127.0.0.1:0', $errno, $errstr);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$errno'. (undefined)

(UnusedLocalVariable)